iset.mm

```diff
@@ -86457,6 +86630,46 @@ finite and infinite sets (and therefore if the set size is a nonnegative
     ( cxnn0 wcel cn0 wn cpnf wceq wo wi elxnn0 pm2.53 sylbi imp ) ABCZADCZEZAFG
     ZNOQHPQIAJOQKLM $.
 
+  ${
+    $d f i $.
+    $( The point at infinity in ` NN+oo ` .  (Contributed by Jim Kingdon,
+       14-Jul-2022.) $)
+    infnninf $p |- ( _om X. { 1o } ) e. NN+oo $=
+      ( vi com c1o csn cxp xnninf wcel c2o cmap csuc cfv wss wral 1oex fvconst2
+      vf cv wceq syl fveq1 co wf sucid df-2o eleqtrri fconst6 elexi elmap mpbir
+      2onn omex peano2 eqtr4d eqimss sseq12d ralbidv df-nninf elrab2 mpbir2an
+      rgen ) BCDEZFGVAHBIUAZGZAQZJZVAKZVDVAKZLZABMZVCBHVAUBBCHCCJHCNUCUDUEUFHBV
+      AHBUJUGUKUHUIVHABVDBGZVFVGRVHVJVFCVGVJVEBGVFCRVDULBCVENOSBCVDNOUMVFVGUNSU
+      TVEPQZKZVDVKKZLZABMVIPVAVBFVKVARZVNVHABVOVLVFVMVGVEVKVATVDVKVATUOUPPAUQUR
+      US $.
+  $}
+
+  ${
+    $d N f i j $.
+    $( Elements of ` NN+oo ` corresponding to natural numbers.  (Contributed by
```

Comment:
Add a second sentence for clarity:
> In other words, the element in NN+oo corresponding to the natural number i is the sequence which "becomes zero" at index i (which explains that the sequence which "never becomes zero", that is, the constant sequence equal to 1, corresponds to infinity, see ~ infnninf ).

Reply:
Agreed this is worth stating. Does the following look clear?

```
    $( Elements of ` NN+oo ` corresponding to natural numbers.  The natural
       number ` N ` corresponds to a sequence of ` N ` ones followed by zeroes.
       Contrast to a sequence which is all ones as seen at ~ infnninf .
       (Contributed by Jim Kingdon, 14-Jul-2022.) $)
```

Reply:
I liked the phrase "becomes zero at index N" (or maybe "switcthes to zero at index N"), but I'll also go with your wording. Then you can prove $\sum_{i \in \mathbb{N}} I(n)_i = n$, and say in a comment that the inverse of I (corestricted to the image of xNN0) is given by summation.